app/digests/tests/test_digest_api.py

```diff
@@ -0,0 +1,8 @@
+from django.test.client import Client
+import pytest
+
+
+@pytest.mark.django_db
+def test_can_get_article_list_xml(client: Client):
+    response = client.get('/digests')
+    assert response.status_code == 200
```

Comment:
should test and implement this hardcoded `Content-Type` header https://github.com/elifesciences/api-raml/blob/develop/src/api.raml#L798

Reply:
should also test for basic keys presence: `items` and `total`

Reply:
Have now added this via a custom paginator as the defaults for `drf` are `count` and `results` 😅